server/utils/format.ts

```diff
@@ -20,6 +20,28 @@ export default {
     return formatted.replace(' at ', ', ')
   },
 
+  /**
+   * Format time of day on Date as Europe/London.
+   *
+   * Example: `11:00 on 22 February 2022`
+   */
+  timeOnDate(date: Date): string {
```

Comment:
why do we need a new date format? i don't think that we should show dates in a different way on every page

Reply:
I am with Igor on this one, I would guess the design is probably shown a different format "by accident"?